Assets/_BForBoss/_Character/Scripts/InputConfiguration.cs

```diff
@@ -0,0 +1,59 @@
+using Perigon.Utility;
+using UnityEngine;
+
+namespace Perigon.Character
+{
+    public class InputConfiguration : IInputConfiguration
```

Comment:
The configuration is soley for storing and getting input configurations from PlayerPrefs. 
Whoever needs it - UI, PlayerMovement, can just create a `new InputConfiguration` and it'll have all the latest stored configs.